daemon/logger/gcplogs/gcplogging_linux.go

```diff
@@ -11,17 +11,15 @@ import (
 // ensureHomeIfIAmStatic ensure $HOME to be set if dockerversion.IAmStatic is "true".
 // See issue #29344: gcplogs segfaults (static binary)
 // If HOME is not set, logging.NewClient() will call os/user.Current() via oauth2/google.
-// However, in static binary, os/user.Current() leads to segfault due to a glibc issue that won't be fixed
-// in a short term. (golang/go#13470, https://sourceware.org/bugzilla/show_bug.cgi?id=19341)
+// If compiling statically, make sure osusergo build tag is also used to prevent a segfault
+// due to a glibc issue that won't be fixed in a short term
+// (see golang/go#13470, https://sourceware.org/bugzilla/show_bug.cgi?id=19341).
 // So we forcibly set HOME so as to avoid call to os/user/Current()
 func ensureHomeIfIAmStatic() error {
 	// Note: dockerversion.IAmStatic and homedir.GetStatic() is only available for linux.
```

Comment:
There is no longer `homedir.GetStatic()`

Reply:
@kolyshkin Should I move ensureHomeIfIAmStatic to gcplogging.go ?

Reply:
Nevermind i'm just removing `GetStatic` from the comment.